.github/workflows/changeset.yml

```diff
@@ -36,26 +36,30 @@ jobs:
               - '!core/chainlink.Dockerfile'
             contracts:
               - contracts/**/*.sol
+              - '!contracts/**/*.t.sol'
             core-changeset:
               - added: '.changeset/**'
             contracts-changeset:
               - added: 'contracts/.changeset/**'
       - name: Make a comment
-        uses: unsplash/comment-on-pr@ffe8f97ccc63ce12c3c23c6885b169db67958d3b # v1.3.0
+        uses: thollander/actions-comment-pull-request@fabd468d3a1a0b97feee5f6b9e499eab0dd903f6 # v2.5.0
```

Comment:
I haven't seen this action yet. Are we using it in any other repos? If not, I know we're using [peter-evans/create-or-update-comment](https://github.com/peter-evans/create-or-update-comment) [elsewhere](https://github.com/smartcontractkit/ea-framework-js/blob/0e0f27e4dd81f4e77f2ba4db623c60df98743163/.github/workflows/add-or-validate-labels.yaml#L60) which would make me prefer that to a new action.

Reply:
hmm I don't think we are but it seems like the `create-or-update-comment` action requires two steps to check for duplicate comment and upsert it whereas this one is taken care of in one step.